backend/app/controllers/spree/admin/reports/total_sales_controller.rb

```diff
@@ -10,6 +10,9 @@ def show
             format.json do
               render json: Spree::Admin::Reports::TotalSales::JsonSerializer.new.call(orders)
             end
+            format.csv do
+              render plain: Spree::Admin::Reports::TotalSales::CsvSerializer.new.call(orders)
```

Comment:
Does that trigger download in a browser, or it merely renders it in a window?

Reply:
yup, triggers download